pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -113,6 +113,7 @@ let
             sed -i "$mf" -e 's|/usr/bin/||g ; s|/bin/||g ; s|/sbin/||g'
         done
         sed -i Makefile -e 's|= depmod|= ${buildPackages.kmod}/bin/depmod|'
+        sed -i Makefile -e 's|--build-id|--build-id=none|'
```

Comment:
Would appreciate a comment here explaining what this line is for.

Reply:
:+1: done